app/build.gradle

```diff
@@ -632,3 +640,17 @@ downloadLicenses {
     includeProjectDependencies = true
     dependencyConfiguration = 'compile'
 }
+
+task registerServiceProviders {
```

Comment:
Why add it at build time instead of just having a static file ?

Reply:
@shubham1g5 that was the initial approach but I thought about making it easy for future additions and also to put this on others radar, a static file can very easily be overlooked. So, not necessarily strong reasons.

Reply:
Reverted